hb_report/utillib.py

```diff
@@ -1632,7 +1633,7 @@ def tail(n, indata):
 
 
 def test_ssh_conn(addr):
-    cmd = r"ssh %s -T -o Batchmode=yes %s true" % (constants.SSH_OPTS, addr)
+    cmd = r"ssh -i %s %s -T -o Batchmode=yes %s true" % (SSH_KEY_CRMSH, constants.SSH_OPTS, addr)
```

Comment:
Use `SSH_WITH_KEY` and `format` to align with other code

Reply:
OK, changed